content/blog/esp32-bootstrapping/index.md

```diff
@@ -191,32 +191,80 @@ typedef struct esp_image_load_header {
 
 ### Memory utilization
 
-Bellow is the image which illustrates how the application linking is done. Highlighted are details of ROM code usage in SRAM1, and I-Cache, D-Cache placement.
-The illustration is covering an single CPU scenario and does not cover the AMP case.
+Following images illustrates how the application linking is done. Highlighted are details of ROM code usage in SRAM. I-Cache and D-Cache allocations are highlighted with red color.
+Important linker symbols are in green bubbles. 
+
+The images below illustrate the process of application linking, focusing on how the ROM code interacts with SRAM. Key details, such as I-Cache and D-Cache allocations, are highlighted in red for clarity.
+Important linker symbols, crucial to understanding the memory layout, are shown in green bubbles.
+These visuals are designed to make the relationships between different components and memory allocations easier to follow.
+
+
+#### ESP32-S3 use-case
+
+Here we are using the ESP32-S3 as reference platform to illustrate the memory utilization, which is **suitable as an reference for majority of newer SoCs**.
+
+- Yellow area is the memory used by the 2nd stage bootloader and can be re-claimed by the application code run-time.
+- Orange area is the memory used by the 1st stage bootloader (or ROM-loader), that can be re-claimed after the application loading is done.
+- Red area is the memory that is not available for the user and should not be used by the linker to spill the code or data.
+
+The following picture illustrates the memory utilization for an single CPU scenario.
+
+{{< figure
+    default=true
+    src="img/esp32s3-zephyr-memory-default.webp"
+    alt=""
+    caption="The ESP32-S3 'default' memory utilization."
+    >}}
+
 
-- Yellow hatched area is the memory that can't be rewritten during the 2nd stage loading (it can be re-claimed during the runtime)
-- Orange hatched area is the memory that can be re-claimed after the application loading is done.
-- Red hatched areas are the memory parts that are not available for the user and should be avoided by the linker.
+The following picture illustrates the memory utilization for a multi CPU scenario.
 
 {{< figure
     default=true
-    src="img/esp32s3-zephyr-memory-usage.webp"
+    src="img/esp32s3-zephyr-memory-amp.webp"
     alt=""
-    caption="The ESP32-S3 memory utilization."
+    caption="The ESP32-S3 'AMP' memory utilization."
     >}}
 
+
 ***NOTE:***
 *The I-Cache allocation SRAM blocks (0,1) are set by the **`PMS_INTERNAL_SRAM_ICACHE_USAGE`** bits
 and the D-Cache allocation SRAM blocks (9,10) are set by the **`PMS_INTERNAL_SRAM_DCACHE_USAGE`** bits, both in the register `PMC_INTERNAL_SRAM_USAGE_1_REG`*
 
+
+#### ESP32 use-case
+
+Here is the memory utilization for the ESP32 platform as its memory model is significantly different from other SoCs in the ESP line.
```

Comment:
_ESP line_ is something new. Let's stick to either _Espressif SoCs_ or _ESP chips_?

```suggestion
Here is the memory utilization for the ESP32 platform as its memory model is significantly different from other Espressif SoCs.
```